draft-ietf-core-object-security.md

```diff
@@ -489,7 +489,7 @@ The support for Observe {{RFC7641}} with OSCORE targets the requirements on forw
 
 Inner Observe SHALL be used to protect the value of the Observe option between the endpoints. Outer Observe SHALL be used to support forwarding by intermediary nodes. 
 
-The server SHALL include a new Partial IV in responses (with or without the Observe option) to Observe registrations.
+The server SHALL include a new Partial IV in responses (with or without the Observe option) to Observe registrations, except for the first response where Partial IV MAY be omitted.
 
```

Comment:
If there is a term such as 'reflecting the request IV' that is consistently used in the document?  if so then using it here would make sense.  If no such term exists then ignore this.

Reply:
No, but we use the sentence "using the nonce from the request". I don't think such a formulation would fit here though.